.github/workflows/fpb-lint.yml

```diff
@@ -14,7 +19,26 @@ jobs:
       with:
         node-version: '16.x'
     - run: npm install -g free-programming-books-lint
-    - run: fpb-lint ./books/
-    - run: fpb-lint ./casts/
-    - run: fpb-lint ./courses/
-    - run: fpb-lint ./more/
+    - run: |
+        fpb-lint ./books/ 2>> error-log.txt | echo error-log.txt
+    - run: |
+        fpb-lint ./casts/ 2>> error-log.txt | echo error-log.txt
+    - run: |
+        fpb-lint ./courses/ 2>> error-log.txt | echo error-log.txt
+    - run: | 
+        fpb-lint ./more/ 2>> error-log.txt | echo error-log.txt
+
+    - run: |
+        gh pr review $PR -a
+        if: success() &&
```

Comment:
Take care about indentations into YAML config files :wink:

Reply:
Said that...

```suggestion
      if: success() &&
```

Reply:
fixed by creating a step